src/mscorlib/src/System/Globalization/CompareInfo.cs

```diff
@@ -170,12 +176,20 @@ public static CompareInfo GetCompareInfo(String name)
 
         public static unsafe bool IsSortable(char ch)
         {
+            if (CultureData.InvariantMode)
+            {
+                return true;
+            }
             char *pChar = &ch;
             return IsSortable(pChar, 1);
         }
 
         public static unsafe bool IsSortable(string text)
         {
+            if (CultureData.InvariantMode)
```

Comment:
will move it after the validation. there is no common working method. we have forked implementation for different OS's